app/frontend/packs/components/checkbox_search_filter.js

```diff
@@ -0,0 +1,202 @@
+const outerHeight = (el) => {
+  let height = el.offsetHeight
+  const style = window.getComputedStyle(el)
+
+  height += parseInt(style.marginTop) + parseInt(style.marginBottom)
+  return height
+}
+
+class CheckboxSearchFilter {
+  constructor (params) {
+    this.params = params
+    this.container = params.container
+    this.container.classList.add('app-checkbox-filter--enhanced')
```

Comment:
Do you know if this is by luck being polyfilled so that it works in IE9, for example?
https://caniuse.com/?search=classlist

Reply:
https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/vendor/polyfills/Element/prototype/classList.js